hibernate-core/src/main/java/org/hibernate/type/format/jakartajson/JsonBJsonFormatMapper.java

```diff
@@ -46,6 +51,9 @@ public <T> T fromString(CharSequence charSequence, JavaType<T> javaType, Wrapper
 
 	@Override
 	public <T> String toString(T value, JavaType<T> javaType, WrapperOptions wrapperOptions) {
+		if ( value == null ) {
```

Comment:
The `value` should never be `null`. Please move this check to the use-sites and ensure this method is never called with a `null` value.